kolibri/deployment/default/settings/base.py

```diff
@@ -132,6 +138,31 @@
 
 USE_TZ = True
 
+Q_CLUSTER = {
```

Comment:
Add a double line break and then section header content above this (see other parts of file for the pattern).